imagetest/test_suites/sql/setup.go

```diff
@@ -28,67 +29,68 @@ const (
 
 // TestSetup sets up the test workflow.
 func TestSetup(t *imagetest.TestWorkflow) error {
```

Comment:
We can skip the test instead if the image isn't windows.

Reply:
For other tests where there's nothing applicable to the image we've been avoiding creating the image at all rather than skipping, to keep results publish to the test grid limited to stuff that's relevant to test.

Reply:
That makes sense. Thanks for clarifying.